app/react/Layout/specs/DocumentCounter.spec.tsx

```diff
@@ -33,5 +34,9 @@ describe('DocumentCounter', () => {
       expect(component.children().get(6).props.children[1]).toBe(100);
       expect(component.children().get(8).props.children).toBe('documents');
     });
+
+    it('should show a + sign next to the total entities when maxedTotalEntities has the gte flag', () => {
+      expect(component.children().get(6).props.children).toContain('+');
+    });
```

Comment:
By passing the 'gte' as the only option in the props, there is no test to prove that the `+` is ever missing.

If you just returned a '+' instead of the logic on the component, all the tests here would be green as well.

That is why following TDD's guidelines is so useful.  In this case, you first create this test.  See it fail, and, what should have happened was to implement the minimum code to make it green: return the '+'.

Then, in order to implement further functionality, you would have required the missing test of: 'when maxedTotalEntities is NOT gte'.

Please, remove the 'gte' from the sent values in props, implement a test proves that only the correct amount is shown, and then implement a test that sends the `gte` with this expect.

But more importantly, please, use TDD on this simpler issues so that it becomes second nature to your programming flow. Otherwise, it is never going to happen.